src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  try {
```

Comment:
try-catch is not for this purpose
if you have a condition to throw an error, you throw it, and then you catch it, decide what to do, how your program will proceed, etc.

here, you do not throw anything in try-block